suites/quincy/rbd/tier-2_rbd_upgrade_5x_to_6x.yaml

```diff
@@ -195,3 +195,24 @@ tests:
             dummy: true
       polarion-id: CEPH-10468
       desc: Verification of image sync continuation after multiple rbd mirror daemon restarts
+
+  - test:
```

Comment:
@sunilangadi2 
nothing apart from Adding after the upgrade test as the suite is being underutilized
IMO, its okay to regress *these* TCs in one of the releases, regressing in both releases not required.

Please let me know if you think otherwise.

Reply:
I think we have rbd mirror regression test suite already in-place for pacific and quincy, And if this test not actually required to run only after the upgrade, Lets keep upgrade test suite for upgrade test cases only.